src/zzz_od/application/intel_board/intel_board_app.py

```
@@ -119,15 +120,48 @@ def close_filter(self) -> OperationRoundResult:
+        stars_list = []
+        try:
+            stars_template = TemplateMatcher(self.ctx.template_loader).match_template(
+                source=self.last_screenshot,
+                template_sub_dir='intel_board',
+                template_id='Star',
+                threshold=0.8,
+                only_best=False
+            )
+            stars_list = [match_result.rect for match_result in stars_template.results]
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)